binnacle/lib/providers/navigation_provider.dart

```diff
@@ -1,65 +1,284 @@
 import 'dart:async';
-
 import 'package:flutter/foundation.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 import 'package:rxdart/rxdart.dart';
+import 'dart:math';
+import 'package:vector_math/vector_math.dart';
 import 'package:sail_routing_dart/route_model.dart';
-import 'package:sos/enums.dart';
+import 'package:sail_routing_dart/polar_plotting/polar_plot.dart';
+import 'package:sail_routing_dart/polar_algs/polar_router.dart';
+import 'package:sail_routing_dart/shared/route_math.dart';
 import 'package:sos/models/position_model.dart';
-import 'package:vector_math/vector_math.dart';
+import 'package:sos/models/ideal_heading_model.dart';
+import 'package:sos/models/wind_model.dart';
+import 'package:sos/enums.dart';
+import 'package:flutter/services.dart' show rootBundle;
 
+const metersPerDegree = 111111;
```

Comment:
The units for these should all be documented. Additionally, I think we might want to move these into the class so they can be modified during runtime.

Reply:
metersPerDegree is an absolute conversion constant, so should not be changed. The others are just defaults, but the ultimate values can be changed during run time via setters.

Reply:
Just added documentation for getters and setters.